sigal/writer.py

```diff
@@ -37,10 +39,8 @@
     os.path.abspath(os.path.dirname(__file__)), 'themes'))
 
 
-class Writer(object):
-    """Generate html pages for each directory of images."""
-
-    template_file = 'index.html'
+class AbstractWriter(object):
+    __metaclass__ = abc.ABCMeta
```

Comment:
`__metaclass__` is a Python 2 thing, with Python 3 you have to use `class AbstractWriter(metaclass=ABCMeta):`. So it is surprising that it works. But anyway this is not needed, I would prefer the class attribute version.